bigquery/google/cloud/bigquery/table.py

```diff
@@ -1026,7 +1026,7 @@ def _configure_job_metadata(metadata,  # pylint: disable=too-many-arguments
         load_config['writeDisposition'] = write_disposition
 
 
-def _parse_schema_resource(info):
+def parse_schema_resource(info):
```

Comment:
We've had [hygeine issues](https://github.com/GoogleCloudPlatform/google-cloud-python/issues/1059) in the past where we expended effort to *remove* such usage.